usr/src/test/zfs-tests/tests/functional/cli_root/zfs_receive/zfs_receive_010_pos.ksh

```diff
@@ -137,6 +137,11 @@ $TOUCH $mntpnt2/f18
 $RM $mntpnt/h17
 $RM $mntpnt2/h*
 
+# Add empty objects to $fs to exercise dmu_traverse code
+for i in `seq 1 100`; do
```

Comment:
On Fri, Dec 18, 2015 at 12:13 PM, Dan McDonald <notifications@github.com>
wrote:

> In
> usr/src/test/zfs-tests/tests/functional/cli_root/zfs_receive/zfs_receive_010_pos.ksh
> <https://github.com/openzfs/openzfs/pull/37#discussion_r48058593>:
>
> > @@ -137,6 +137,11 @@ $TOUCH $mntpnt2/f18
> >  $RM $mntpnt/h17
> >  $RM $mntpnt2/h*
> >
> > +# Add empty objects to $fs to exercise dmu_traverse code
> > +for i in `seq 1 100`; do
>
> Some distros do for virtual methadone to Linux users. OmniOS has it in
> /bin and in /usr/gnu/bin. If a native illumos seq(1) ever happened, the
> /bin/ would be replaced in favor of its native version.
>

​Thinking about ways to restrict the set of utilities that can be run by
the suite, I'm interested in knowing more about why we object to seq. Is it
simply because it's part of a GNU package? If so, why? Is it because brace
expansion is more efficient? Are there other reasons?​

-- 
John Wren Kennedy

275 Middlefield Road, Suite 50
Menlo Park, CA 94025
http://www.delphix.com/